src/adv2_file.cpp

```diff
@@ -925,6 +925,13 @@ ADVRESULT Adv2File::GetFrameImageSectionHeader(int streamId, int frameId, unsign
 			// Skip 1 byte of streamId, 16 bytes of start and end timestamps and 4 bytes of section size
 			advfseek(m_Adv2File, 1 + 16 + 4, SEEK_CUR);
 
+			// TODO
```

Comment:
You are quite right Andreas - good catch. The spec indeed says the streamid should be repeated. This is unnecessary and as you rightly say doing so will break a lot of existing .adv files. So I will update the spec, not the code.